### DISCUSSION TITLE
JFrog offline?

### DISCUSSION THREAD

**Discussion by Milner99** [winder/Universal-G-Code-Sender#1866] [Help] [answered]
Hi All,

Anyone know of another download source for UGS - JFrog seems to be offline.

Or can help me build after cloning the repo.

Thank you

**Comment by breiler** [comment, marked as answer]
Hi,

@winder has tried to make contact with JFrog, hopefully the situation will be resolved soon. In the meantime you can use the temporary links from here (under assets): https://github.com/winder/Universal-G-Code-Sender/releases/tag/v2.0.11

I noticed that I forgot to upload the "raspberry pi"-build for ARM based linux. It should now also be available.

**Comment by cnbaker** [comment]
22 days and still down?!?! Maybe it's time to switch platforms...

**Reply by breiler** [reply]
What makes you say it is still down? Works fine for me.

**Reply by cnbaker** [reply]
It must be intermittent because when I tried it (over the course of several hours) prior to commenting, the download site was offline. It does appear to be working for me now, however.
